### DISCUSSION TITLE
The font-family is distorted in nextjs15

### DISCUSSION THREAD

**Discussion by Wilian-Pereira-W** [vercel/next.js#70231] [Help]
### Link to the code that reproduces this issue

https://codesandbox.io/p/devbox/dark-tdd-26fhps?file=%2Fapp%2Flayout.tsx%3A13%2C23

### To Reproduce

Just check the text on the page. Just check the text on the page. Happens in dev, build and production

### Current vs. Expected behavior

It was supposed to appear, "não". But it's looking like the picture
![letra](https://github.com/user-attachments/assets/ce30ce4d-ef0b-4160-aaf4-bb4f56f20e90)

### Provide environment information

```bash
"next": "15.0.0-canary.155",
"react": "19.0.0-rc-206df66e-20240912",
"react-dom": "19.0.0-rc-206df66e-20240912"
"@types/node": "20.12.12",
"@types/react": "18.3.3",
"@types/react-dom": "18.3.0",
"typescript": "5.3.3"
```

### Which area(s) are affected? (Select all that apply)

Font (next/font)

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local)

**Comment by samcx** [comment]
@Wilian-Pereira-W I wasn't able to replicate this. →

![CleanShot 2024-09-16 at 15 46 28@2x](https://github.com/user-attachments/assets/4fe7d80e-81bf-4b43-8daa-75ac181d78be)

![CleanShot 2024-09-16 at 15 47 08@2x](https://github.com/user-attachments/assets/2edcc7e5-edc3-44ea-ae4e-a6c813223378)

**Comment by Wilian-Pereira-W** [comment]
Here continues the reported behavior.
![bug](https://github.com/user-attachments/assets/36a243d8-49a6-4f83-9ad3-32bc87fb6b19)

**Comment by samcx** [comment]
Can you see if it happens locally for you and not on CodeSandbox?

**Comment by eduardodallmann** [comment]
@Wilian-Pereira-W  I can't reproduce the problem here either. I suggest testing on other browsers and another computer.
![image](https://github.com/user-attachments/assets/bdea6b62-8dc9-406a-b9c4-e0b9400c52cd)
